mail/templates/product_order_receipt/body.html

```diff
@@ -31,7 +31,7 @@ <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px;
                 <strong style="font-weight: 700;">Unit Price:</strong> ${{ line.price }}<br>
                 <strong style="font-weight: 700;">Discount:</strong> ${{ line.discount }}<br>
                 {% if enable_taxes_display %}
-                    <strong style="font-weight: 700;">Tax:</strong> ${{ line.tax_paid|floatformat:2 }}<br>
+                    <strong style="font-weight: 700;">Tax ({{ order.tax_rate|floatformat }}%):</strong> ${{ line.tax_paid|floatformat:2 }}<br>
```

Comment:
I tried that and then removed it because it prints the zeros as well and that didn't look good to me. And it only made sense that we should ideally not be expecting the tax rates to go that long in decimals. 

_An example of above thought_: If we use `floatformat:2` and we have a tax rate of `18.0000` (This is how we store it in DB, With 4 decimal precision). In this case, it would always be `18.00%`. The extra zeros here really felt unnecessary. Although it works fine for actual numbers e.g. (`18.12`) as decimals precision for others it doesn't look really good.

I see a couple of solutions for more sanity:
1. Either we store the tax rate as 2 decimal places in DB.
2. Now that I have taken a deeper look, I saw [this doc from Django on `floatformat` tag](https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#floatformat) I believe we should use `floatformat:"-2"` instead which should do exactly what we want.

I'll update the PR with solution#2 mentioned above when I address the feedback. 

@asadali145 Let me know what you think.

Reply:
Sounds good to me.